.travis.yml

```diff
@@ -12,11 +12,11 @@ branches:
 
 jobs:
     include:
-        - name: "Build with most recent GCC & Clang, run tests"
```

Comment:
it still the most recent gcc AFAIK

Reply:
Yes, but then I'd have to say the same about clang, and mentioning it didn't feel as necessary to me anymore. Contrasted with the ancient GCC, it's quite obvious that we're using recent toolchains.

Reply:
Makes sense